edit/cms_functions.php

```diff
@@ -0,0 +1,478 @@
+<?php
+
+/*
+ * Gets a PDO object to use for database queries. This PDO's fetch mode is set to FETCH_ASSOC by default.
+ *
+ * @Return returns the PDO object to use for our given queries.
+ */
+function getPDO() {
+    $db = new PDO('mysql:dbname=CMS;host=127.0.0.1','root');
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    return $db;
+}
+
+/*
+ * Displays the form for editing the About Me section of the portfolio. The value for each input is filled automatically with the current entry
+ * in the database.
+ *
+ * @return Returns the input elements for the About Me form, formatted as HTML. This should be placed inside a form element, before the submit
+ * element.
+ */
+function displayAboutMeInput() {
```

Comment:
probably a little over engineered having this code as a function, you could just put it into the html file. no point changing it now but it would simply a over complicated project

Reply:
Is this sort of thing what you mean by 'complicated architecture'?